app/graphql/types/profile_stats/anime_amount_consumed.rb

```diff
@@ -0,0 +1,8 @@
+class Types::ProfileStats::AnimeAmountConsumed < Types::BaseObject
+  implements Types::ProfileStats::ProfileStatInterface
+
+  field :data, Types::ProfileStats::AmountConsumed,
+    null: false,
+    description: 'The breakdown of this specific stat.',
+    method: :stats_data
```

Comment:
Is there a reason for this, versus using an interface like `implements Types::ProfileStats::AmountConsumed`?

Reply:
🤔 no, interface is most def the right way to go.